configs/trainer/default.yaml

```diff
@@ -1,5 +1,8 @@
 _target_: pytorch_lightning.Trainer
 
+default_root_dir: ${paths.output_dir}
+enable_model_summary: False
```

Comment:
what is `enable_model_summary`? why is this added within this PR?

Reply:
This was a part of the PR, but maybe not required wrt current changes.

Reply:
please have a look what this parameter does and then decide, if it makes sense to include it.

Reply:
This parameter was eventually removed in https://github.com/ashleve/lightning-hydra-template/pull/358 . Not sure why it was even introduced at first place.

Reply:
ok, then let's remove it directly